helpers/event/event_test_creator.py

```diff
@@ -71,3 +71,49 @@ def createPastEvent(self, only_event=False):
             mtc = MatchTestCreator(event)
             mtc.createCompleteQuals()
         return event
+
```

Comment:
There's no need to create the test events here, since they're not used anywhere else. Just create the Event objects within `test_event.py`